src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -173,7 +182,20 @@ class SeekBar extends Slider {
    *         The percentage of media played so far (0 to 1).
    */
   getPercent() {
-    const percent = this.getCurrentTime_() / this.player_.duration();
+    const currentTime = this.getCurrentTime_();
+    let percent;
+    const liveTracker = this.player_.liveTracker;
+
+    if (liveTracker.isLive()) {
+      percent = (this.player_.currentTime() - liveTracker.seekableStart()) / liveTracker.liveWindow();
```

Comment:
could you just use `getCurrentTime_ ` here?

Reply:
yeah we should